NEWS.md

```diff
@@ -1,3 +1,6 @@
+# 1.9.2 2022-06-28
```

Comment:
```suggestion
# 1.10.0 2022-06-28
```

Reply:
For next iteration it should be 1.10.0 xxxx-xx-xx
But for this release 1.9.2 2022-06-28